File: lista 2/cpp/navio.cpp
#include <iostream>
using namespace std;
int main(){
    int X, Y;
    cin>>X>>Y;
    if((X <= 0 or X >= 100) | (Y <= 0 or Y >= 100)){
        cout<<"Coordenada invalida";
        
    }
    else{
        if((X>=71 or Y>=71)){
            cout<<"Coordenada valida e o navio esta longe";
        }
        else{
            cout<<"Coordenada valida e o navio esta perto";
        }
    }    
}

File: lista 2/python/navio.py
X,Y = map(int, input().split())
if(X <= 0 or X >= 100) or (Y <= 0 or Y >= 100):
  print("Coordenada invalida")
else:
  if(X>=71 or Y>=71):
    print("Coordenada valida e o navio esta longe")
  else:
    print("Coordenada valida e o navio esta perto")

